components/apimgt/org.wso2.carbon.apimgt.persistence/src/test/java/org/wso2/carbon/apimgt/persistence/utils/PersistenceHelper.java

```diff
@@ -215,7 +215,7 @@ public static GenericArtifact getSampleAPIProductArtifact() throws GovernanceExc
         artifact.setAttribute("overview_visibleTenants",null);
         artifact.setAttribute("overview_name","APIProductTest");
         artifact.setAttribute("overview_apiSecurity","oauth2,oauth_basic_auth_api_key_mandatory,basic_auth,api_key");
-        artifact.setAttribute("overview_environments","Production and Sandbox");
+        artifact.setAttribute("overview_environments","Default");
```

Comment:
this property as well

Reply:
The underlying method and class is used only in test cases. So this attribute cannot be removed.